website/intro/use-cases.html.markdown

```diff
@@ -87,11 +72,8 @@ allows the network to better support the applications running on top. Most SDN
 implementations have a control layer and infrastructure layer.
 
 Terraform can be used to codify the configuration for software defined networks.
-This configuration can then be used by Terraform to automatically setup and modify
-settings by interfacing with the control layer. This allows configuration to be
-versioned and changes to be automated. As an example, [AWS VPC](https://aws.amazon.com/vpc/)
-is one of the most commonly used SDN implementations, and [can be configured by
-Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc).
+This configuration can then be used by Terraform to automatically setup and modify settings by interfacing with the control layer. This allows configuration to be
+versioned and changes to be automated. As an example, [AWS VPC](https://aws.amazon.com/vpc/) is one of the most commonly used SDN implementations, and [can be configured by Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc).
```

Comment:
Judith said I should review content in a separate PR to separate concerns, so that's why I didn't adjust this copy. (I did the initial "what is Terraform" page before we had that convo). But I do agree with you! Let me go back in and edit this page because I agree it needs a refresh.